packages/react-saasify/src/components/ServiceForm/ReactFromJSONSchema.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import ReactFromJSON from 'react-from-json'
+
+const mapProp = (prop, onChange) => {
+  if (prop && prop !== onChange && prop.type) {
+    const { type, ...props } = prop
+
+    return {
+      type,
+      props: {
+        ...props,
+        onChange,
+        name: props.propKey
+      }
+    }
+  }
+
+  return prop
+}
+
+export const ReactFromJSONSchema = ({ definition, mapping, onChange }) => (
+  <ReactFromJSON
+    entry={{ ...definition.params.schema.properties, type: 'entry' }}
```

Comment:
Some of these schemas will have a ton of properties, but only a few that are interesting / required.q

I'd suggest either filtering these to only required properties and/or setting a max limit on the number of properties.

Reply:
Currently the only use case gets restricted only to props in the example, so not sure we need to limit at this point.

Reply:
Ahhh didn't realize this. Good call :)